### DISCUSSION TITLE
v5.5.0

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16483] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.5.0'>v5.5.0</a>.</em>

**Comment by lubber-de** [comment]
Windows build for 8.1?

Why was the windows build for PHP 8.1 dropped? This was offered until phalcon 5.4.0?
Of course we need even more support for php 8.2/8.3, but why is 8.1. now also dropped? We have lots of working windows machines using php 8.1 and cannot upgrade to 8.2/8.3 because phalcon builds are not offered, but 8.1 was working, so what's the reason to drop it for phalcon 5.5.0?

**Comment by HiralRajgor** [comment]
phalcon 5.2 was working fine on alpine linux and now updated automatically to 5.5 now idk some of the working code is giving 502 bad gateway, it is not priting any error in nginx logs nor php or phalcon error logs. @niden  Please help me out, it is production code of my working project
